scrapers_next/ia/people.py

```diff
@@ -53,21 +59,40 @@ def process_page(self):
         except SelectorError:
             pass
 
-        table = XPath("//div[@class='legisIndent divideVert']//td//text()").match(
+        other_info = XPath("//div[@class='legisIndent divideVert']//table").match(
             self.root
-        )
-
-        # the fields, like "cell phone", etc. are located at every odd indice
-        # the information for each field, like the phone number, are located at every even indice
-        fields = list(map(self.get_field, table[0::2]))
-        extra = table[1::2]
-
-        num_of_fields = range(len(fields))
-
-        for i in num_of_fields:
-            if fields[i] == "Legislative Email":
-                continue
-            p.extras[fields[i].lower()] = extra[i].strip()
+        )[0]
+
+        if len(other_info.getchildren()) > 1:
+            info_rows = XPath("//tr").match(other_info)
+            for row in info_rows:
+                if len(row) > 1:
+                    raw_field_name = row[0].text_content().strip().lower()
+                    field_name = re.sub(":", "", raw_field_name)
+                    field_text = row[0].text_content().strip()
```

Comment:
There's one issue with this line that's evident looking at the scraped data

Reply:
Indexing [fixed](https://github.com/openstates/openstates-scrapers/pull/4240/commits/f392aad64f0976b08a3e729b487e813f4ec31cf1#diff-5a784d04c8f85a0f44a3f26120c302cbb53f79a668d824d66fb8c6f45dc4e443R72) on that line.